app/assets/stylesheets/components/_filter-panel.scss

```diff
@@ -0,0 +1,48 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-panel__content {
+  background-color:  govuk-colour("light-grey");
+  padding: govuk-spacing(3);
+  margin-top: govuk-spacing(2);
+}
+
+.app-c-filter-panel__header {
+  display: flex;
+  flex-wrap: wrap;
+  place-content: space-between;
+  align-items: center;
+  gap: govuk-spacing(2);
+
+  .gem-c-heading {
+    font-weight: normal;
+  }
+
+  .app-c-filter-panel__button {
```

Comment:
Do we want this button to act more like a link? I think so but not 100% sure. In which case we can probably use something like this on this class:

```scss
cursor: pointer;
@include govuk-link-decoration;

&:hover {
  color: $govuk-link-hover-colour;
  @include govuk-link-hover-decoration;
}
```